ocs_ci/deployment/install_ocp_on_rhel.py

```diff
@@ -69,6 +70,9 @@ def __init__(self, rhel_worker_nodes):
 
         self.ocp_repo = get_ocp_repo()
 
+        # Approve the pending CSRs
+        approve_pending_csr()
```

Comment:
If we will update logic here: https://github.com/red-hat-storage/ocs-ci/pull/1938/files to check each node has approved CSR we should not do it here and we can close this PR.